### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by lesidon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+My name is <ins>Leonid</ins>. Here I can do some nice things with ```markdown```: e.g this beautiful <i>equation</i>: h<sub>&theta;</sub>(x) = &theta;<sub>o</sub> x + &theta;<sub>1</sub>x
+
+I was born in the city called Ryazan and this is a beatiful one (find a picture below) :smile:
+
+![alt text](https://img.geliophoto.com/ryazan/00_ryazan.jpg)
```
Comment (okeowo-cloud) [inline README.md]: It's such a beautiful city!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+My name is <ins>Leonid</ins>. Here I can do some nice things with ```markdown```: e.g this beautiful <i>equation</i>: h<sub>&theta;</sub>(x) = &theta;<sub>o</sub> x + &theta;<sub>1</sub>x
```
Comment (nvaulin) [inline README.md]: heh, good equation!

File: README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+I was born in the city called Ryazan and this is a beatiful one (find a picture below) :smile:
+
+![alt text](https://img.geliophoto.com/ryazan/00_ryazan.jpg)
+
+
+
+Actually, you could find my twitter account, if you press another picture below 
+
+
+<a href="https://mobile.twitter.com/leagronom"><img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS4n_urpJ9XpwOTdzBVbGvactwHrPagYQrTJPYjxfxLGkSyu7nJZVqRVGAeohnPgKMrnKE&usqp=CAU" alt="Five cats looking around a field."/></a>
```
Comment (nvaulin) [inline README.md]: yeahh, html!!

**Review by nvaulin — COMMENTED** [review]
For now you can merge the pull-request to have this great readme in _main_ branch. Then you will have it at your GitHub profile page!

**Pull request closed without merging**